rss2email/feed.py

```diff
@@ -207,20 +207,26 @@ class Feed (object):
         'digest_post_process',
         ]
 
-    def __init__(self, name=None, url=None, to=None, config=None):
+    def __init__(self, name=None, url=None, to=None, from_email=None, config=None):
         self._set_name(name=name)
         self.reset()
         self.__setstate__(dict(
                 (attr, getattr(self, attr))
                 for attr in self._dynamic_attributes))
         self.load_from_config(config=config)
+        if from_email:
+            self.from_email = from_email
         if url:
             self.url = url
         if to:
             self.to = to
```

Comment:
I understand that the code already was like this, but why on earth would you conditionally set fields like this?
This is going to lead to badly debuggable crashes!

```
In [1]: class Foo():      
   ...:     def __init__(self, bar=None):
   ...:         if bar:
   ...:             self.bar = bar

In [3]: Foo(bar="foo").bar
Out[3]: 'foo'

In [4]: Foo().bar
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-c643d71cfbb3> in <module>()
----> 1 Foo().bar

AttributeError: Foo instance has no attribute 'bar'

Reply:
I see, I didn’t read the code correctly then (to be honest I didn’t look at the beginning of `__init__`.